uber/templates/regform.html

```diff
@@ -91,6 +91,14 @@
         });
     {% endif %}
 
+    $(function () {
+        staffingClicked();
+        if ($.field('staffing')) {
+            {% if attendee.shifts %} $.field('staffing').prop('readonly', true).prop('title', 'Please {{ "contact your department head to drop shifts" if c.AT_THE_CON else "unassign yourself from shifts" }} before changing your volunteer status.'); {% endif %}
```

Comment:
Adding this line back in still presents a problem, because it's _supposed_ to only be 'readonly' for public-facing pages but currently will make the checkbox readonly on admin pages as well.